admin-dev/themes/default/template/controllers/login/content.tpl

```diff
@@ -53,7 +53,7 @@
 	<div class="flip-container">
 		<div class="flipper">
 			<div class="front front_login panel" {if isset($reset_token) && isset($id_employee)}style="display:none;"{/if}>
-				<h4 id="shop_name">{$shop_name}</h4>
+				<h4 id="shop_name" style="max-height: 1em">{$shop_name|truncate:200:"...":true}</h4>
```

Comment:
If you used `style` I guess it's because you could not use CSS? 😢

Reply:
He could, there is a stylesheet for this

Reply:
Sorry it was a WIP PR ... I had a discuss with Valentin to find a better solution